versioned_docs/version-3.0.0-beta/billing/coupon.md

```diff
@@ -0,0 +1,39 @@
+---
+id: coupon
+title: Coupon Activation
+sidebar_label: Coupon
+slug: /billing/coupon
+---
+
+Spheron automatically provides all users with a free Starter package to access and use the platform when you sign up until the user upgrades to the Pro plan.
+
+To check your spheron Plan [**please click here**](https://docs.spheron.network/billing/plans)
+
+
+## How to upgrade your plan using the Coupon code options
```

Comment:
Can you add another section for activating coupons from the coupon page by clicking add coupon?

Reply:
@rekpero  please explain this further.

Reply:
You can activate coupon by following these step as well:
- Go to billing page, then click coupon tab
- Then click on **Add new coupon** button
- You will get a popup to enter your coupon code, once you fill the coupon code, click on Add button.

I would like you to mention these steps with screenshot in this coupon page in doc